org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/editor/quickfix/XtextGrammarQuickfixTest.xtend

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019 RCP Vision s.r.l. (http://www.rcp-vision.com) and others.
+ * Copyright (c) M�lardalen University (http://www.mdh.se) and others.
```

Comment:
Maybe that was a misunderstanding. You cannot change the original copyright holder, e.g. Mälardalen Uni is subsummized as 'others' in this case.

Reply:
I realized that I had inserted the wrong name and thought I could change it. I will revert the last commit, sorry.